tests/frameworks/OSXShims/include/windows.h

```diff
@@ -0,0 +1,55 @@
+//******************************************************************************
```

Comment:
Can you fix the unit test instances?  

---
In reply to: [72359856](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72359856) [](ancestors = 72359856,72351063,72312429)